### PR TITLE
Add bio for kpineiro

### PR DESCRIPTION
**Pull request by kpineiro**
Pull request for branch kpineiro

### CODE REVIEW

**Comment by kpineiro** [comment]
Check

File: kpineiro.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#KPINEIRO
+
+* Name: Ken Pineiro
+* Username: kpineiro
+* Location: Reston, VA
```
Comment (kpineiro) [inline kpineiro.md:6]: Add fav dessert

**Comment by kpineiro** [comment]
@Sawan12345 Does this look ok?

**Comment by Sawan12345** [comment]
@kpineiro  Sorry I missed your notification somehow, Can you reopen and try adding you pet's name?

**Pull request merged**